src/openrct2/Editor.cpp

```diff
@@ -500,12 +500,9 @@ namespace Editor
 
         for (const auto& parkEntrance : gParkEntrances)
         {
-            int32_t x = parkEntrance.x;
-            int32_t y = parkEntrance.y;
-            int32_t z = parkEntrance.z / 8;
             int32_t direction = direction_reverse(parkEntrance.direction);
 
-            switch (footpath_is_connected_to_map_edge(x, y, z, direction, 0))
+            switch (footpath_is_connected_to_map_edge(parkEntrance, direction, 0))
```

Comment:
CoordsXYZD would be preferable here, I'd say.

Reply:
The direction represents where you want to check where as the xyz represents the footpath tile you are looking from. They shouldn't be tied together.

Reply:
Not changing anything then